[RHACS] [rhacs-docs-4.4] ROX23460 Adding 4.4.1 RN

agantony opened the pull request:
<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OCPBUGS#<jira-issue-id>][OSDOCS#<jira-issue-id>]: <short-description-of-the-pr> --->

<!--- If your changes apply to the latest release and/or in-development version of OpenShift, open your PR against the `main` branch.

* For more details about the information requested in this template, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/create_or_edit_content.adoc#submit-PR --->

Version(s):
4.4
<!--- Specify the version or versions of OpenShift your PR applies to. -->

Issue:
https://issues.redhat.com/browse/ROX-23460
<!--- Add a link to the Bugzilla, Jira, or GitHub issue, if applicable. --->

Link to docs preview:
<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. --->

SME review:
- [ ] SME has approved this change.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

Additional information:
- Merge into `rhacs-docs-4.4`
<!--- Optional: Include additional context or expand the description here.--->

<!--- After you open your PR, ask for review from the OpenShift docs team:
  For community authors: Tag @openshift/team-documentation in a GitHub comment.--->

agantony commented:
/label RHACS

ovalenti commented on release_notes/44-release-notes.adoc:
```diff
@@ -561,6 +562,28 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
+
+*Updated on*: 15 April 2024
+
+* Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where Collector uses the EBPF collection method when CORE_BPF is selected from {product-title-short} 4.4.1 and later.
++
+Workaround: If you use OpenShift Container Platform 4.12 Power offline, you must use a support package.
```
```suggestion
* Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where CORE_BPF collection method is not functioning properly. From {product-title-short} 4.4.1 and later, in this particular case, Collector uses automatically the EBPF collection method when CORE_BPF is selected.

Impact: If you use OpenShift Container Platform 4.12 on the Power architecture and in offline mode, you must use a support package.
```

ovalenti reviewed (changes requested):
I expanded slightly to give more context, and replaced the term `workaround` since this known issue is about the limitations of a workaround; which seemed a bit misleading.

agantony commented:
> I expanded slightly to give more context, and replaced the term `workaround` since this known issue is about the limitations of a workaround; which seemed a bit misleading.

@ovalenti  I have incorporated your suggestions. If the changes look good to you, please approve the PR. Thanks!

ovalenti reviewed (approved):
The "known issues" section looks good to me, thanks 👍 

I read through the other parts and it seems alright, but I cannot validate the content.

dhaus67 commented on release_notes/44-release-notes.adoc:
```diff
@@ -561,6 +562,27 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
+
+* With this update, rendering the Helm charts `stackrox-central-services` and `stackrox-secured-cluster-services` requires a Helm version 3.9.0 or later, as support for older Helm versions has been discontinued.
+
+* Fixed an issue where {product-title-short} would not qualify the registry and path of images from the container runtime if the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` environment variable was set to `true` on both Central and Sensor. This update now enables support for CRI-O's unqualified search registries and short name aliases.
```
I thought we do not document this - @dcaravel to verify whether this should be here or not.

dcaravel replied:
That's correct, please remove the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` section for now. Thanks @agantony

agantony replied:
ack

dhaus67 reviewed (approved):
The rest of the release note section looks good to me; I have one comment regarding the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` - pinged @dcaravel to confirm, please do not merge without his approval.

kcarmichael08 commented on release_notes/44-release-notes.adoc:
```diff
@@ -561,6 +571,23 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
+
+*Updated on*: 15 April 2024
+
+* Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected. 
```
```suggestion
* Currently, there is an issue for OpenShift Container Platform 4.12 on the IBM Power architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected. 
```

kcarmichael08 commented on release_notes/44-release-notes.adoc:
```diff
@@ -561,6 +571,23 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
+
+* Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected. 
+
+If you are using OpenShift Container Platform 4.12 on the Power architecture and in offline mode, you must use a support package.
```
```suggestion
If you are using OpenShift Container Platform 4.12 on the IBM Power architecture and in offline mode, you must use a support package.
```

kcarmichael08 reviewed (commented):
Couple of minor things, otherwise lgtm!

kcarmichael08 commented:
This is ready to merge, but I think we need to wait for Monday 4/15 until the release goes out.
